controls/SV-244531.rb

```diff
@@ -39,16 +34,14 @@
   tag cci: ['CCI-000366']
   tag nist: ['CM-6 b']
 
```

Comment:
in the rhel7 profile this is NA for the container, we should verify if that should be the case here

Reply:
I'd argue that a container shouldn't have interactive shells in the first place. Nobody should be shelling into them in prod.